We don't provide systemd, so don't hide or disable gnome-remote-desktop sharing if systemd is not available.

diff --git a/panels/system/cc-system-panel.c b/panels/system/cc-system-panel.c
--- a/panels/system/cc-system-panel.c
+++ b/panels/system/cc-system-panel.c
@@ -153,15 +153,9 @@
 static void
 cc_system_panel_init (CcSystemPanel *self)
 {
-    CcServiceState service_state;
-
     g_resources_register (cc_system_get_resource ());
     gtk_widget_init_template (GTK_WIDGET (self));
 
-    service_state = cc_get_service_state (REMOTE_DESKTOP_SERVICE, G_BUS_TYPE_SYSTEM);
-    /* Hide the remote-desktop page if the g-r-d service is either "masked", "static", or "not-found". */
-    gtk_widget_set_visible (GTK_WIDGET (self->remote_desktop_row),
-                            service_state == CC_SERVICE_STATE_ENABLED || service_state == CC_SERVICE_STATE_DISABLED);
     gtk_widget_set_visible (GTK_WIDGET (self->software_updates_group), show_software_updates_group (self));
 
     cc_panel_add_static_subpage (CC_PANEL (self), "about", CC_TYPE_ABOUT_PAGE);
diff --git a/panels/system/remote-desktop/cc-desktop-sharing-page.c b/panels/system/remote-desktop/cc-desktop-sharing-page.c
--- a/panels/system/remote-desktop/cc-desktop-sharing-page.c
+++ b/panels/system/remote-desktop/cc-desktop-sharing-page.c
@@ -172,7 +172,6 @@
 
     if (!cc_enable_service (REMOTE_DESKTOP_SERVICE, G_BUS_TYPE_SESSION, &error)) {
         g_warning ("Failed to enable remote desktop service: %s", error->message);
-        disable_gnome_desktop_sharing_service (self);
     }
 }
 
